internal/hammer/diff.go

```diff
@@ -445,7 +445,9 @@ func (g *Generator) generateDDLForColumns(from, to *Table) DDL {
 			continue
 		}
 
-		if fromCol.Hidden != toCol.Hidden {
+		isFromColHidden := !fromCol.Hidden.Invalid() && fromCol.Hidden != token.Pos(0)
+		isToColHidden := !toCol.Hidden.Invalid() && toCol.Hidden != token.Pos(0)
+		if isFromColHidden != isToColHidden {
```

Comment:
To make the condition easier to understand, I think it would be helpful to extract it into a function like `if isColHidden(fromCol) != isColHidden(toCol)`